railties/lib/rails/commands/dbconsole.rb

```diff
@@ -1,13 +1,49 @@
 require 'erb'
 require 'yaml'
 require 'optparse'
+require 'rails/commands/console_helper'
 
 module Rails
   class DBConsole
+    include ConsoleHelper
+
     attr_reader :arguments
 
-    def self.start
-      new.start
+    class << self
+      def parse_arguments(arguments)
```

Comment:
@rafaelfranca because the same method was at class level in ```Console``` class and I believe that this method does nothing about the instance itself so IMHO this should be a class method.